app/controllers/devise_registration_controller.rb

```diff
@@ -197,10 +197,8 @@ def update
     resource_updated = update_resource(resource, account_update_params)
     yield resource if block_given?
     if resource_updated
-      # this is the change to the standard controller method:
```

Comment:
What were these about originally?
Did we change the code in between since we wrote them?

Reply:
We've changed more of the code outside of these comments